utility/src/test/java/org/oppia/android/util/logging/EventBundleCreatorTest.kt

```diff
@@ -1460,6 +1472,262 @@ class EventBundleCreatorTest {
     assertThat(bundle).string("install_id").isEqualTo(TEST_INSTALLATION_ID)
   }
 
+  @Test
+  @RunParameterized(
+    Iteration("forHomeScreen", "screenNameCase=HOME_ACTIVITY", "screenNameCaseValue=home_activity"),
```

Comment:
```suggestion
    Iteration("home", "name=HOME_ACTIVITY", "expStrName=home_activity"),
```

Per my suggestions above. Here, we can fit it easily. Hopefully most can fit on one line now with this approach which should significantly reduce how much space needs to be dedicated for the test.

Reply:
done.